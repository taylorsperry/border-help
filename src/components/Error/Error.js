import React from 'react'

export const Error = () => {
  return (
    <div>
      That page does not exist
    </div>
  )
}

export default Error