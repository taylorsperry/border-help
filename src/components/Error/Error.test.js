import React from 'react'
import { Error } from './Error'
import { shallow } from 'enzyme'

describe('Error', () => {
  let wrapper
  it('should match the snapshot',() => {
    wrapper = shallow(<Error />)
    expect(wrapper).toMatchSnapshot()
  })
})