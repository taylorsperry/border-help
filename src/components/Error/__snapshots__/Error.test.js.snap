// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
<div>
  That page does not exist
</div>
`;
